Makefile.js

```diff
@@ -738,6 +742,7 @@ target.gensite = function(prereleaseVersion) {
                 `title: ${title}`,
                 "layout: doc",
                 `edit_link: https://github.com/eslint/eslint/edit/master/${filePath}`,
+                ruleType,
```

Comment:
This change adds a `ruleType` field in a rule page's frontmatter in case we want to expose that somehow on the page.